.github/CODEOWNERS

```diff
@@ -429,8 +429,6 @@
 #
 #     https://help.github.com/en/articles/about-code-owners
 #
-# On modifying this list make sure to keep the CODEOWNER_IMPORTANT_PATHS list
```

Comment:
Why delete? /cc @DubeySandeep

Reply:
Oh that was unintentional. Sorry for that. Fixed!